NEWS.md

```diff
@@ -1,5 +1,28 @@
 # pkgdown (development version)
 
+* Code blocks in sections (including details) of reference topics are now highlighted depending on the language information, if the package documentation was built with roxygen2 >= 7.1.1.9001. For better results it is important to add language information i.e. the YAML code block below, in [roxygen2 with Markdown support](https://roxygen2.r-lib.org/articles/rd-formatting.html#turning-on-markdown-support-1), would otherwise be parsed as R code.
```

Comment:
Here's a note about the roxygen2 version.

Now the problem is that if you used another version of roxygen2, you just get NO syntax highlighting at all of code blocks in Rd sections.

Reply:
Maybe some different behavior/warning depending on the roxygen2 version from DESCRIPTION?

Reply:
Let me release roxygen2 so we can at least rely on a version on CRAN. Once that's done, I think it's fine to just mention that it needs latest roxygen2.